include/RED4ext/Scripting/Natives/Generated/ex/EntitySpawner.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+// clang-format off
+
+// This file is generated from the Game's Reflection data
+
+#include <cstdint>
+#include <RED4ext/Common.hpp>
+#include <RED4ext/Scripting/Natives/Generated/game/IGameSystem.hpp>
+
+namespace RED4ext
+{
+namespace ex
+{
+struct EntitySpawner : game::IGameSystem
```

Comment:
This class is from CyberEngineTweaks, it shouldn't be in SDK. You must've forgotten to disable all mods when running `Dump()`.